client/src/components/timebutton/index.js

```diff
@@ -4,9 +4,9 @@ import "./timebutton.css";
 
 class Timebutton extends Component {
   render() {
-    const { textvalue } = this.props;
+    const { textvalue, onClick } = this.props;
     return (
-      <button className="time-btn-style" type="button">
+      <button className="time-btn-style" onClick={this.props.onClick}>
```

Comment:
`onClick={onClick}`
You've already destructured it on line 7... use it here.

Reply:
@NoureldeanSaed done